### PR TITLE
Print etcd version when calling openshift version

### PR DESCRIPTION
**Pull request by mfojtik**
Fixes: #5003

### CODE REVIEW

**Comment by mfojtik** [comment]
@smarterclayton PTAL (this only prints version of etcd when called with openshift version)

File: pkg/version/version.go [hunk 1]
```diff
@@ -59,6 +60,9 @@ func NewVersionCommand(basename string) *cobra.Command {
 		Run: func(c *cobra.Command, args []string) {
 			fmt.Printf("%s %v\n", basename, Get())
 			fmt.Printf("kubernetes %v\n", kubeversion.Get())
+			if basename == "openshift" {
```
Comment (smarterclayton) [inline pkg/version/version.go]: This doesn't work - you have to support atomic-enterprise, origin, others.  You should pass a boolean in that says "print etcd" and set that in the caller.

Comment (mfojtik) [inline pkg/version/version.go]: oh I forgot about those...

Comment (liggitt) [inline pkg/version/version.go]: see sketch at https://github.com/openshift/origin/pull/4566/files#diff-bcee38a2cd2e5f1990f046729f3833d8

**Comment by mfojtik** [comment]
@liggitt the does `!IsClient()` guarantee that it is only server? I kind of like @smarterclayton idea of passing the boolean and then setting it on places that calls it.

**Comment by mfojtik** [comment]
@smarterclayton fixed PTAL

File: pkg/version/version.go [hunk 2]
```diff
@@ -52,13 +53,16 @@ func (info Info) String() string {
 }
 
 // NewVersionCommand creates a command for displaying the version of this binary
-func NewVersionCommand(basename string) *cobra.Command {
+func NewVersionCommand(basename string, printEtcdVersion bool) *cobra.Command {
```
Comment (liggitt) [inline pkg/version/version.go]: would we just keep augmenting this with additional flags, e.g. `printClientFeatures bool`?

Comment (smarterclayton) [inline pkg/version/version.go]: Probably

File: pkg/cmd/server/start/kubernetes/kubernetes.go [hunk 3]
```diff
@@ -37,7 +37,7 @@ func NewCommand(name, fullName string, out io.Writer) *cobra.Command {
 	cmds.AddCommand(NewProxyCommand("proxy", fullName+" proxy", out))
 	cmds.AddCommand(NewSchedulerCommand("scheduler", fullName+" scheduler", out))
 	if "hyperkube" == fullName {
-		cmds.AddCommand(version.NewVersionCommand(fullName))
+		cmds.AddCommand(version.NewVersionCommand(fullName, true))
```
Comment (smarterclayton) [inline pkg/cmd/server/start/kubernetes/kubernetes.go]: False, hyperkube doesn't run etcd.

**Comment by mfojtik** [comment]
@smarterclayton fixed.

**Comment by smarterclayton** [comment]
LGTM [merge]

**Comment by nak3** [comment]
Is this just call internal etcd version? If you use external etcd, you get wrong version?

**Comment by liggitt** [comment]
This is the version of the embedded etcd, yes

**Comment by nak3** [comment]
Thank you. I opened as this issue https://github.com/openshift/origin/issues/5563

**Pull request merged**